java-frontend/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -646,4 +653,47 @@ private void checkExplodedGraphTooBig(ProgramState programState) {
     }
   }
 
+  /**
+   * This class ensures that the SE checks are placed in the correct order for the ExplodedGraphWalker
+   * In addition, checks that are needed for a correct ExplodedGraphWalker processing are provided in all cases.
+   *
+   */
+  public static class EGWFactory {
+
+    private final ConditionAlwaysTrueOrFalseCheck alwaysTrueOrFalseChecker;
+    private final List<SECheck> seChecks = new ArrayList<>();
+
+    public EGWFactory(List<JavaFileScanner> scanners) {
+      List<SECheck> seChecks = new ArrayList<>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "seChecks" which hides the field declared at line 664. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AHiddenFieldCheck)